evap/evaluation/templates/contact_modal.html

```diff
@@ -22,18 +24,22 @@ <h5 class="modal-title" id="{{ modal_id }}Label">{{ title }}</h5>
                 <div class="modal-body">
                     {{ teaser }}
                     <br /><br />
-                    <div class="mb-3">
-                        <label for="{{ modal_id }}SenderName" class="control-label">{% trans 'Sender' %}:</label>
-                        <input type="text" class="form-control mb-3" id="{{ modal_id }}SenderName" disabled value="{{ user.full_name }}" />
-                    </div>
-                    <div class="mb-3">
-                        <label for="{{ modal_id }}Subject" class="control-label">{% trans 'Subject' %}:</label>
-                        <input type="text" class="form-control" id="{{ modal_id }}Subject" disabled value="{{ title }}" />
-                    </div>
-                    <div class="mb-3">
-                        <label for="{{ modal_id }}MessageText" class="control-label">{% trans 'Message' %}:</label>
-                        <textarea autofocus class="form-control" id="{{ modal_id }}MessageText" style="resize:vertical;min-height:150px;"></textarea>
+                    <div class="mb-3 modal-grid">
```

Comment:
please use the usual form layout we have e.g. in the evaluation edit form or the delegation modal form with bold labels, no colon and larger margin between label and field.

Reply:
this is now done with .modal-grid .control-label rule in _modal.scss